settings/base.py

```diff
@@ -78,6 +78,10 @@
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
         'APP_DIRS': True,
         'OPTIONS': {
+            'context_processors': [
+                'django.contrib.auth.context_processors.auth',
```

Comment:
Why did these get added?

Reply:
This is needed to allow me to run migrations from within the venv of edx-ora2.